docs/migration-guides/1.10-1.11.md

```diff
@@ -0,0 +1,54 @@
+# Migrate from v1.10 to v1.11
+
+## Knative Eventing Mesh 
+
+To use new [Knative Eventing Mesh](https://kyma-project.io/docs/master/components/knative-eventing-mesh/#overview-overview), you must upgrade Kyma from version 1.10 to version 1.11. To do so, follow the [Kyma upgrade guide](https://kyma-project.io/docs/#installation-upgrade-kyma). 
+
+**CAUTION**: Events sent during the migration can be lost.
+    
+### Verification
+
+1. List all the applications:
+
+    ```bash
+     kubectl get applications 
+    ```  
+2. Check if each Application has a corresponding HTTP Source Adapter. 
+
+    ```bash
+    kubectl -n kyma-integration get pods 
+    ```
+3. Check if the Channel status is `READY`
+
+4. Check if Kyma Subscriptions have been converted to Knative Triggers:
+
+    ```bash
+    kubectl -n test get triggers 
```

Comment:
```suggestion
    kubectl --all-namespaces get triggers 
```

would it make sense to check for remaining subscriptions as well?   
```bash
k get -A subscriptions
```

Reply:
@k15r I would go for the same approach like following comment
`-n <YOUR_NAMESPACE>`